aspnet/tutorials/publish-to-azure-webapp-using-vs.rst

```diff
@@ -1,47 +1,181 @@
-Publish to an Azure Web App using Visual Studio
-===============================================
+Deploy an ASP.NET Core web app to Azure using Visual Studio
+============================================================
 
-By `Erik Reitan`_
+By `Rick Anderson`_
 
-This article describes how to publish an ASP.NET web app to Azure using Visual Studio. 
 
-**Note:** To complete this tutorial, you need a Microsoft Azure account. If you don't have an account, you can `activate your MSDN subscriber benefits`_ or `sign up for a free trial`_.
+.. contents:: Sections:
+  :local:
+  :depth: 1
 
-.. _`activate your MSDN subscriber benefits`: http://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/?WT.mc_id=A261C142F
+Set up the development environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. _`sign up for a free trial`: http://azure.microsoft.com/pricing/free-trial/?WT.mc_id=A261C142F
+- `Install the latest Azure SDK for Visual Studio 2015 <http://go.microsoft.com/fwlink/?linkid=518003>`__. The SDK installs Visual Studio 2015 if you don't already have it.
 
+.. note:: The SDK installation can take more than 30 minutes if your machine doesn't have many of the dependencies.
 
-Start by either creating a new ASP.NET web app or opening an existing ASP.NET web app. 
+- Install `.NET Core + Visual Studio tooling <http://go.microsoft.com/fwlink/?LinkID=798306>`__
 
-1. In **Solution Explorer** of Visual Studio, right-click on the project and select **Publish**.
+- Verify your `Azure account <https://portal.azure.com/>`__. You can `open a free Azure account <https://azure.microsoft.com/pricing/free-trial/>`__ or `Activate Visual Studio subscriber benefits <https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/>`__.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/01-Publish.png
+Create a web app
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-2. In the **Publish Web** dialog box, click on **Microsoft Azure Web Apps** and log into your Azure subscription.
+From the Visual Studio **Start** page, tap **New Project**.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/02-PublishWebdb.png
+.. image:: first-mvc-app/start-mvc/_static/new_project.png
 
-3. Click **New** in the **Select Existing Web App** dialog box to create a new Web app in Azure.
+Alternatively, you can use the menus to create a new project. Tap **File > New > Project**.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/03-SelectExistingWebAppdb.png
+.. image:: first-mvc-app/start-mvc/_static/new_project.png
```

Comment:
This is the same screen shot as the one above. I think you meant new_project2.png.